operations/agent-flow-mixin/dashboards/opentelemetry.libsonnet

```diff
@@ -0,0 +1,174 @@
+        panel.withPosition({ x: 0, y: 0, w: 8, h: 10 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr=|||
+              rate(receiver_accepted_spans_ratio_total{cluster="$cluster", namespace="$namespace", instance=~"$instance"}[$__rate_interval])
+            |||,
+            //TODO: How will the dashboard look if there is more than one receiver component? The legend is not unique enough?
+            legendFormat='{{ pod }} / {{ transport }}',
+          ),
+        ])
+      ),
+      (
+        panel.new(title='Refused spans', type='timeseries') +
+        stackedPanelMixin +
+        panel.withDescription(|||
+          Number of spans that could not be pushed into the pipeline.
+        |||) +
+        panel.withPosition({ x: 8, y: 0, w: 8, h: 10 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr=|||
+             rate(receiver_refused_spans_ratio_total{cluster="$cluster", namespace="$namespace", instance=~"$instance"}[$__rate_interval])
+            |||,
+            legendFormat='{{ pod }} / {{ transport }}',
+          ),
+        ])
+      ),
+      (
+        panel.newHeatmap('RPC server duration (traces)') +
+        panel.withUnit('milliseconds') +
+        panel.withDescription(|||
+          The duration of inbound RPCs.
+        |||) +
+        panel.withPosition({ x: 16, y: 0, w: 8, h: 10 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='sum by (le) (increase(rpc_server_duration_milliseconds_bucket{cluster="$cluster", namespace="$namespace", instance=~"$instance", rpc_service="opentelemetry.proto.collector.trace.v1.TraceService"}[$__rate_interval]))',
+            format='heatmap',
+            legendFormat='{{le}}',
+          ),
+        ])
+      ),
+
+      // "Batching" row
+      (
+        panel.new('Batching [otelcol.processor.batch]', 'row') +
+        panel.withPosition({ h: 1, w: 24, x: 0, y: 10 })
+      ),
+      (
+        panel.newHeatmap('Number of units in the batch') +
+        panel.withDescription(|||
+          Number of units in the batch
+        |||) +
+        panel.withPosition({ x: 0, y: 10, w: 8, h: 10 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr='sum by (le) (increase(processor_batch_batch_send_size_ratio_bucket{cluster="$cluster", namespace="$namespace", instance=~"$instance"}[$__rate_interval]))',
+            format='heatmap',
+            legendFormat='{{le}}',
+          ),
+        ])
+      ),
+      (
+        panel.new(title='Distinct metadata values', type='timeseries') +
+        panel.withDescription(|||
+          Number of distinct metadata value combinations being processed
+        |||) +
+        panel.withPosition({ x: 8, y: 10, w: 8, h: 10 }) +
+        panel.withQueries([
+          panel.newQuery(
+            expr=|||
+             sum(processor_batch_metadata_cardinality_ratio{cluster="$cluster", namespace="$namespace", instance=~"$instance"}) or vector(0)
```

Comment:
As discussed offline: this probably makes more sense per-pod, similar to other queries instead of an aggregated metric.